kf_lib_data_ingest/target_apis/kids_first.py

```diff
@@ -182,8 +182,8 @@
             }
         ],
         'properties': {
-            'external_sample_id': CONCEPT.BIOSPECIMEN.UNIQUE_KEY,
-            'external_aliquot_id': CONCEPT.BIOSPECIMEN.ALIQUOT_ID,
+            'external_sample_id': CONCEPT.BIOSPECIMEN_GROUP.UNIQUE_KEY,
```

Comment:
```suggestion
            'external_sample_id': CONCEPT.BIOSPECIMEN_GROUP.ID,
```

Reply:
Why not unique key?

Reply:
Because that's not what unique key is for

Reply:
The dataservice _does_ need a field for what unique key is doing, but external ID isn't it

Reply:
unique keys should basically never go to the dataservice right now until it has a place to put our shenanigans that doesn't affect the portal